pkgs/by-name/gr/gridix/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  wrapGAppsHook3,
+  gtk3,
+  openssl,
+  xdotool,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gridix";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "MCB-SMART-BOY";
+    repo = "Gridix";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
https://github.com/NixOS/nixpkgs/blob/master/doc/build-helpers/fetchers.chapter.md#fetchgit-fetchgit

tag is preferred nowadays